infrastructure/terraform/etc/env_eu-west-2_prod.tfvars

```diff
@@ -0,0 +1,28 @@
+environment  = "prod"
+account_name = "notify-reporting-prod"
+
+core_account_id = "746418818434"
+core_env        = "prod"
+
+# PowerBI On-Premises Gateway variables:
+enable_powerbi_gateway = true
+
+public_subnet_cidrs  = [
+  "10.0.1.0/24",
+  "10.0.2.0/24",
+  "10.0.3.0/24"
+]
+
+private_subnet_cidrs = [
+  "10.0.4.0/24",
+  "10.0.5.0/24",
+  "10.0.6.0/24"
+]
+
+instance_type    = "t3.medium"
```

Comment:
guessing we might change this later?

Reply:
Yes - this is just a workable instance type for now. To be reviewed under actual loads. Can also cluster potentially.